docs/nginx-ingress-controllers.md

```diff
@@ -14,7 +14,7 @@ The table below summarizes the key difference between nginxinc/kubernetes-ingres
 | --- | --- | --- | --- |
 | **Fundamental** |
 | Authors | Kubernetes community | NGINX Inc and community |  NGINX Inc and community |
-| NGINX version | [Custom](https://github.com/kubernetes/ingress-nginx/tree/master/images/nginx) NGINX build that includes several third-party modules | NGINX official mainline [build](https://github.com/nginxinc/docker-nginx) | NGINX Plus |
```

Comment:
could you revert changes in this file? the links are for a different repo

Reply:
fixed